src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/validator.html.twig

```diff
@@ -0,0 +1,98 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% if collector.data.nb_violations > 0 or collector.data.calls|length %}
+        {% set status_color = collector.data.nb_violations ? 'red' : '' %}
+        {% set icon %}
+            {{ include('@WebProfiler/Icon/validator.svg') }}
+            <span class="sf-toolbar-value">
+                {{ collector.data.nb_violations ?: collector.data.calls|length }}
```

Comment:
I agree with @xabbuh here. It looks weird to switch between violation and calls.